test/rules/beta/test_marginals.jl

```diff
@@ -0,0 +1,20 @@
+module RulesBetaMarginalsTest
+
+using Test
+using ReactiveMP
+using Random
+import ReactiveMP: @test_marginalrules
+
+@testset "marginalrules:Beta" begin
+
+     @testset "m_out, m_a, m_b: (m_out::Beta, m_a::PointMass, m_b::PointMass)" begin
```

Comment:
Can you please change title to reflect marginal name `:out_a_b`.
Something like 
```
@testset ":out_a_b (m_out::Beta, m_a::PointMass, m_b::PointMass)" begin
...
end
```